submissions/webdevtoday/html-css-popup/css/style.css

```diff
@@ -0,0 +1,203 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+:root {
+    --icon-text-color: rgb(117, 117, 117);
+    --icon-outline-color: rgb(117, 117, 117);
+    --nav-popup-border-color: #e3e3e3;
+    --nav-popup-bg-color: #fff;
+    --nav-popup-height: 520px;
+}
+
+html {
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 16px;
+}
+
+ul li {
+    list-style: none;
+    display: block;
+}
+
+img {
+    display: block;
+    width: 100%;
+}
+
+.main-nav {
+    max-width: 1366px;
+    max-height: 80px;
+}
+
+.main-nav__container {
+    display: flex;
+    align-items: center;
+    justify-content: flex-end;
+    padding: 12px 10px;
+}
+
+.main-nav__item {
+    position: relative;
+    display: block;
+    min-width: 32px;
```

Comment:
Explain to me please why do you need these `min-width` and `min-height` here?

Reply:
I need it because I want to control size of items of nav menu.

Reply:
Does it affect your layout? Because when I removed this `min-width: 32px;` and ` min-height: 32px;` nothing happened.

Reply:
Delete it?

Reply:
If it doesn't affect your layout, you can remove these redundant styles.

Reply:
Ok, I'll delete it.

Reply:
I've done it.